main/lsp/AutocorrectFlusher.cc

```diff
@@ -0,0 +1,22 @@
+#include "main/lsp/AutocorrectFlusher.h"
+
+using namespace std;
+namespace sorbet::realmain::lsp {
+
+void AutocorrectFlusher::flushErrors(spdlog::logger &logger, vector<unique_ptr<core::ErrorQueueMessage>> errors,
+                                     const core::GlobalState &gs, core::FileRef file) {
+    for (auto &error : errors) {
+        if (error->error->isSilenced) {
```

Comment:
You need to check that `error->kind` is `Error`, otherwise this may cause a bug. (While this should not actually happen, it could theoretically be a `Flush` or a `Query`, and it's best to program defensively.)